docs/reference/indices/warmers.asciidoc

```diff
@@ -182,7 +182,12 @@ Getting a warmer for specific index (or alias, or several indices) based
 on its name. The provided name can be a simple wildcard expression or
 omitted to get all warmers. 
 
+ifdef::asciidoctor[]
+added[1.4.0.Beta1,"The API will always include a `warmers` section, even if there aren't any warmers. Previous versions would not return the `warmers` section"]
```

Comment:
Could you do `added::[` here?

Reply:
Added with 7557afe260fde429989b86c35344aae40746d018. Thank @nik9000 .